examples/sos_decomposition.jl

```diff
@@ -0,0 +1,50 @@
+using DynamicPolynomials
+using SumOfSquares
+using JuMP
+using SCS
+
+# ------------------------------------------------------------------------------
+# A trivial SOS decomposition example
+# ------------------------------------------------------------------------------
+
+# The polynomial p = x^2 - x*y^2 + y^4 + 1 is SOS.
+# Among infinite others, it has the decomposition:
```

Comment:
What do you mean by "Among infinite others" ? Isn't it the only polynomial with this decomposition ?